mmv1/third_party/terraform/services/datafusion/resource_data_fusion_instance_test.go

```diff
@@ -123,6 +126,7 @@ resource "google_data_fusion_instance" "foobar" {
   name   = "%s"
   region = "us-central1"
   type   = "ENTERPRISE"
+  deletion_protection = false
```

Comment:
no need to set.... it is by default false.

Reply:
done